### PR TITLE
roadmap fixes

### PR DESCRIPTION
**Pull request by AconiteGodOfSWG**

### CODE REVIEW

File: sku.0/sys.server/compiled/game/script/library/gcw.java [hunk 1]
```diff
@@ -2691,6 +2691,10 @@ public static boolean gcwIsInvasionCityOn(String city) throws InterruptedExcepti
             CustomerServiceLog("gcw_city_invasion", "gcw.gcwIsInvasionCityOn: GCW City: " + city + " is not configured to run a city invasion. Function returning False.");
             return false;
         }
+        if (city == "dearic" && Boolean.parseBoolean(getConfigSetting("GameServer", "lifeday")) == true) {
```
Comment (Gene64) [inline sku.0/sys.server/compiled/game/script/library/gcw.java:2694]: Should probably use `city.equals("dearic")` to compare the two strings.

Comment (Gene64) [reply sku.0/sys.server/compiled/game/script/library/gcw.java:2694]: Also you should probably use the `checkConfigFlag()` method from the utils class instead.

**Comment by IrishDarkshadow** [comment]
Cekis, could you please have a look at this one and see what action should be taken?

**Comment by Cekis** [comment]
Looks good to me.  I agree with Ios that's the better way.

**Pull request merged**